thoth/user_api/payload_filter.py

```diff
@@ -65,6 +66,14 @@ def process(self, webhook_payload: dict) -> dict:
                 elif payload.get("action") == "removed":
                     self._remove_event(payload.get("repositories_removed"))
                     return None
+            if event == "push":
+                # Do not re-run kebechet if push event is on bot branch.
+                ref = payload.get("ref")
+                if ref.startswith(_BOT_BRANCH):
```

Comment:
Could `ref` be `None` here? In that case, we will error out with an internal error.

Reply:
Ideally it shouldn't be in any event from Github, but I have added a null check just in case :)